lib/manifold/api/workspace.rb

```diff
@@ -17,6 +17,10 @@ def initialize(name, template_path: DEFAULT_TEMPLATE_PATH, logger: Logger.new($s
         @vector_service = Services::VectorService.new(logger)
       end
 
+      def self.from_directory(directory, logger: Logger.new($stdout))
+        new(directory.basename.to_s, logger: logger)
```

Comment:
Minor point, but unless you expect this to run on Ruby 3.0 or earlier, you can use shorthand hash syntax here.
```ruby
new(directory.basename.to_s, logger:)
```

Reply:
I just checked your pipeline and your test matrix starts at Ruby 3.1.6, so I would go all-in. Set **TargetRubyVersion** in **.rubocop.yml** to 3.1, **required_ruby_version** to "**>= 3.1.0**" in **manifold.gemspec**, and run:
```sh
bundle exec rubocop --only Style/HashSyntax --autocorrect`
```
Alternatively, if you really want to support Ruby 3.0, then you should add it to your test matrix.

Reply:
Speaking of RuboCop, you should enable new cops:
```yaml
AllCops:
  NewCops: enable
```